src/bidiMapper/domains/network/NetworkStorage.ts

```diff
@@ -135,12 +133,10 @@ export class NetworkStorage {
   }
 
   deleteRequest(id: Network.Request) {
-    const request = this.#requestMap.get(id);
-    if (request) {
-      request.dispose();
+    if (this.#requestMap.has(id)) {
```

Comment:
Default maps are tricky as they will create the resources every time.
We should only dispose them when we actually have them.